### PR TITLE
Switch to Gnome Platform 47 runtime

### PR DESCRIPTION
**Pull request by akurtakov**

### CODE REVIEW

**Comment by Mailaender** [comment]
The loading screen is a small gray rectangle and this error message (not sure if related)

```
*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug

*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug

*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug

*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug
```

Otherwise works fine.

File: org.eclipse.Java.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
         "--socket=x11",
```
Comment (Mailaender) [inline org.eclipse.Java.json:15]: Remove this one then.

Comment (akurtakov) [reply org.eclipse.Java.json]: Done

**Comment by akurtakov** [comment]
> The loading screen is a small gray rectangle and this error message (not sure if related) appears in console
> 
> ```
> *** BUG ***
> In pixman_region32_init_rect: Invalid rectangle passed
> Set a breakpoint on '_pixman_log_error' to debug
> 
> *** BUG ***
> In pixman_region32_init_rect: Invalid rectangle passed
> Set a breakpoint on '_pixman_log_error' to debug
> 
> *** BUG ***
> In pixman_region32_init_rect: Invalid rectangle passed
> Set a breakpoint on '_pixman_log_error' to debug
> 
> *** BUG ***
> In pixman_region32_init_rect: Invalid rectangle passed
> Set a breakpoint on '_pixman_log_error' to debug
> ```
> 
> Otherwise works fine.

I don't believe this is flatpak related as I have seen it in the logs in my unzipped eclipse install.

**Review by Mailaender — APPROVED** [review]
Tested under Wayland. The crashes mentioned in https://github.com/flathub/org.eclipse.Java/pull/95 are gone so this is save now.

**Pull request merged**